pulse2percept/api.py

```diff
@@ -1,444 +1,486 @@
-
-        Parameters
-        ----------
-        stim : utils.TimeSeries|list|dict, optional
-            An input stimulus, as passed to p2p.pulse2percept. If given,
-            activated electrodes will be highlighted in the plot.
-            Default: None
-        ax : matplotlib.axes._subplots.AxesSubplot, optional
-            A Matplotlib axes object. If None given, a new one will be created.
-            Default: None
-
-        Returns
-        -------
-        Returns a handle to the created figure (`fig`) and axes element (`ax`).
-        """
-        from matplotlib import patches
-
-        self._set_layers()
-
-        fig = None
-        if ax is None:
-            # No axes object given: create
-            import matplotlib.pyplot as plt
-            fig, ax = plt.subplots(1, figsize=(10, 8))
-
-        ax.set_facecolor('black')
-        ax.plot(self.ofl.jan_x[:, ::5], -self.ofl.jan_y[:, ::5],
-                c=(0.5, 1, 0.5))
-
-        # Draw in the the retinal patch we're simulating.
-        # This defines the size of our "percept" image below.
-        dva_xmin = retina.ret2dva(self.ofl.gridx.min())
-        dva_ymin = -retina.ret2dva(self.ofl.gridy.max())
-        patch = patches.Rectangle((dva_xmin, dva_ymin),
-                                  retina.ret2dva(self.ofl.range_x),
-                                  retina.ret2dva(self.ofl.range_y),
-                                  alpha=0.7)
-        ax.add_patch(patch)
-
-        # Highlight location of stimulated electrodes
-        if stim is not None:
-            for key in stim:
-                el = self.implant[key]
-                if el is not None:
-                    ax.plot(retina.ret2dva(el.x_center),
-                            -retina.ret2dva(el.y_center), 'oy',
-                            markersize=np.sqrt(el.radius) * 2)
-
-        # Plot all electrodes and their label
-        for e in self.implant.electrodes:
-            ax.text(retina.ret2dva(e.x_center + 10),
-                    -retina.ret2dva(e.y_center + 5),
-                    e.name, color='white', size='x-large')
-            ax.plot(retina.ret2dva(e.x_center),
-                    -retina.ret2dva(e.y_center), 'ow',
-                    markersize=np.sqrt(e.radius))
-
-        ax.set_aspect('equal')
-        ax.set_xlim(-20, 20)
-        ax.set_xlabel('visual angle (deg)')
-        ax.set_ylim(-15, 15)
-        ax.set_ylabel('visual angle (deg)')
-        ax.set_title('Image flipped (upper retina = upper visual field)')
-        ax.grid('off')
-
-        return fig, ax
-
-
-def get_brightest_frame(percept):
-    """Returns the brightest frame of a percept
-
-    This function returns the frame of a percept (brightness over time) that
-    contains the brightest pixel.
-
-    Parameters
-    ----------
-    percept : TimeSeries
-        The brightness movie as a TimeSeries object.
-    """
-    _, frame = percept.max_frame()
-
-    return frame
+import numpy as np
```

Comment:
Yeah, for some reason this file had some non-Unix line-endings, and when I fixed them the file suddenly showed up as changed all over...